src/components/UploadAvatar/UploadAvatar.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState, React } from 'react';
+import { Avatar, Input } from '@chakra-ui/react';
+import { AddIcon, Icon } from '@chakra-ui/icons';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'AddIcon' is defined but never used.